src/functionalTest/resources/application-functional.yaml

```diff
@@ -1,16 +1,15 @@
-targetInstance: ${TEST_URL:http://rd-professional-api-aat.service.core-compute-aat.internal}
+targetInstance: ${TEST_URL:http://rd-professional-api-pr-243.service.core-compute-preview.internal}
 s2s-url: ${S2S_URL_FOR_TESTS:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
 
 s2s-name: rd_professional_api
 
 idam.api.url: ${IDAM_URL:https://idam-api.aat.platform.hmcts.net}
 generated.user.email.pattern: ${GENERATED_USER_EMAIL_PATTERN:freg-test-user-%s@feemail.com}
-test.url: ${TEST_URL:http://rd-professional-api-aat.service.core-compute-aat.internal}
 test.user.password: ${TEST_USER_PASSWORD:Hmcts123}
-#oauth2.redirect.uri: ${OAUTH2_REDIRECT_URI:https://rd-professional-api-preview.service.core-compute-preview.internal/oauth2redirect}
 oauth2.client.id: ${OAUTH2_CLIENT_ID:rd-professional-api}
 oauth2.client.secret: ${OAUTH2_CLIENT_SECRET:cc5f2a6-9690-11e9-bc42-526af7764f64}
 idam.auth.clientId: rd-professional-api
+idam.auth.clientSecret: cc5f2a6-9690-11e9-bc42-526af7764f64
```

Comment:
Always use env variables. And NEVER the aat (sensitive) hard-coded defaults.

Reply:
Please remove

Reply:
For this separate JIRA has raised and will be fixed under that.